README.md

```diff
@@ -129,154 +131,3 @@ Both options require you generate an SSL certificate and a key via OpenSSL:
 ```sh
 openssl req -x509 -newkey rsa:2048 -keyout key.pem -out cert.pem -days 365
 ```
-
```

Comment:
going to move all the parts below this into a docs.mrjs.io page (or as an add on to the existing ECS page there) and link to that up here

Reply:
related to https://github.com/Volumetrics-io/documentation/pull/35